diff --git a/includes/config.php b/includes/config.php
index a71ad78..f100eef 100644
--- a/includes/config.php
+++ b/includes/config.php
@@ -4,13 +4,13 @@
 	//----------------------------------------------------------------------------------//
 	
 	/*  Set the URL to your Sendy installation (without the trailing slash) */
-	define('APP_PATH', 'http://your_sendy_installation_url');
+	define('APP_PATH', getenv('APP_PATH'));
 	
 	/*  MySQL database connection credentials (please place values between the apostrophes) */
-	$dbHost = ''; //MySQL Hostname
-	$dbUser = ''; //MySQL Username
-	$dbPass = ''; //MySQL Password
-	$dbName = ''; //MySQL Database Name
+	$dbHost = getenv('MYSQL_HOST'); //MySQL Hostname
+	$dbUser = getenv('MYSQL_USER'); //MySQL Username
+	$dbPass = getenv('MYSQL_PASSWORD'); //MySQL Password
+	$dbName = getenv('MYSQL_DATABASE'); //MySQL Database Name
 	
 	
 	//----------------------------------------------------------------------------------//	
